### PR TITLE
qt5: update 5.6.0

### PR DESCRIPTION
**Pull request by pullmoll**
See #3853

### CODE REVIEW

**Comment by pullmoll** [comment]
As it is now `xbps-install -S -R /data/void-glibc/binpkgs/qt5.6 qt5` does not work.

The missing packages `qt5-doc`, `qt5-webkit{,-devel}`, `qt5-quick1{,-devel}` prevent this.

I'm working on separate `qt5-doc` and `qt5-webkit{,-devel}` packages but `qt5-quick1` is definitely gone. How do I remove it without the need to rebuild every(?) dependency on `qt5`?

**Comment by pullmoll** [comment]
From my point of view the PR is now completed. I could update `qt5` with my locally built packages for `x86_64` and pkgs depending on `qt5` continue to work as expected.

I'd be happy if someone would scrutinize the changes and merge, if everything seems ok.

**Comment by xtraeme** [comment]
:+1:

File: srcpkgs/qt5-doc/template [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Template file for 'qt56-doc'
```
Comment (Vaelatern) [inline srcpkgs/qt5-doc/template]: May as well be updated to be the same as the pkgname.

**Comment by pullmoll** [comment]
Actually there are some duplicate files left in the qt5 packages. A complete list is here: http://sprunge.us/deKU

The functions `_install_subpkg()` and `_install_devel()` need to `rm` some more files. I'll take a look at what's left that should not be.

File: srcpkgs/qt5-doc/template (hunk 1 above)
Comment (pullmoll) [inline srcpkgs/qt5-doc/template]: Good catch :)

**Comment by pullmoll** [comment]
Duplicate files are gone, all seems fine now.

**Comment by Gottox** [comment]
**Never ever** remove (sub-)packages if you don't know *exactly* what you are doing. What happened:

1. qtcreator-full depends on qt5-quick1-devel
2. this PR removed qt5-quick1-devel but didn't update the corresponding packages that depend on that package
3. This:

```
# sudo xbps-install -u 
MISSING: qt5-script-5.5.1_4
Transaction aborted due to unresolved dependencies.
```

**Comment by pullmoll** [comment]
D'OH! I didn't think about that. I should have checked for anyone using qt5-quick1.

The qt5-quick1 is definitely gone and non pkg really linked against it - this was what I verified and what wasn't sufficient.

**Comment by Gottox** [comment]
maybe it's time for a xbps-src check-consistency :)

**Pull request merged**